src/Modules/Finance/FinanceLayout.module.css

```diff
@@ -0,0 +1,3 @@
+.FinanceLayout {

```

Comment:
I don't wanna nitpick but what's the reason for naming classes with a capital letter?

Reply:
To make them seem more coherent to the elements they are applied to. This is actually the only reason.

Reply:
I guess, the best practice for us is to establish the common rules of naming classes

Reply:
@Pavel-Yarakhovich @DariaYarohovich so let's use lower-case in class naming and I'll try to add some rules for css in prettier